jea/role-capabilities.md

```diff
@@ -157,42 +185,56 @@ FunctionDefinitions = @{
 
-The body (script block) of custom functions runs in the default language mode for the system and is not subject to JEA's language constraints.
-This means that functions can access the file system and registry, and run commands that were not made visible in the role capability file.
-Take care to avoid allowing arbitrary code to be run when using parameters and avoid piping user input directly into cmdlets like `Invoke-Expression`.
+The body (script block) of custom functions runs in the default language mode for the system and
+isn't subject to JEA's language constraints. This means that functions can access the file system
+and registry, and run commands that weren't made visible in the role capability file. Take care to
+avoid running arbitrary code when using parameters. Avoid piping user input directly into cmdlets
+like `Invoke-Expression`.
 
-In the above example, you will notice that the fully qualified module name (FQMN) `Microsoft.PowerShell.Utility\Select-Object` was used instead of the shorthand `Select-Object`.
-Functions defined in role capability files are still subject to the scope of JEA sessions, which includes the proxy functions JEA creates to constrain existing commands.
+In the above example, notice that the fully qualified module name (FQMN)
+`Microsoft.PowerShell.Utility\Select-Object` was used instead of the shorthand `Select-Object`.
+Functions defined in role capability files are still subject to the scope of JEA sessions, which
+includes the proxy functions JEA creates to constrain existing commands.
 
-Select-Object is a default, constrained cmdlet in all JEA sessions that doesn't allow you to select arbitrary properties on objects.
-To use the unconstrained Select-Object in functions, you must explicitly request the full implementation by specifying the FQMN.
-Any constrained cmdlet in a JEA session will exhibit the same behavior when invoked from a function, in line with PowerShell's [command precedence](/powershell/module/microsoft.powershell.core/about/about_command_precedence).
+By default, `Select-Object` is a constrained cmdlet in all JEA sessions that doesn't allow the
+selection of arbitrary properties on objects. To use the unconstrained `Select-Object` in functions,
+you must explicitly request the full implementation using the FQMN. Any constrained cmdlet in a JEA
+session has the same constraints when invoked from a function. For more information, see
+PowerShell's [command precedence](/powershell/module/microsoft.powershell.core/about/about_command_precedence).
 
-If you are writing a lot of custom functions, it may be easier to put them in a [PowerShell Script Module](/powershell/developer/windows-powershell).
-You can then make those functions visible in the JEA session using the VisibleFunctions field like you would with built-in and third party modules.
+If you're writing several custom functions, it's simpler to put them in a [PowerShell Script Module](/powershell/developer/windows-powershell).
```

Comment:
Link text: Windows PowerShell SDK

Reply:
I removed the link. I don't think we need it here.